store/task_run.go

```diff
@@ -61,7 +61,7 @@ func (raw *taskRunRaw) toTaskRun() *api.TaskRun {
 }
 
 // createTaskRunImpl creates a new taskRun.
-func (s *Store) createTaskRunImpl(ctx context.Context, tx *sql.Tx, create *api.TaskRunCreate) (*taskRunRaw, error) {
```

Comment:
Use (*Store) if possible, similar to the controller in tests.

Reply:
SG.